### PR TITLE
chore/CircleCI setup

### PR DESCRIPTION
**Pull request by vanyavasylyshyn**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      - checkout
+      - run:
+          name: Install dependencies
+          command: |
+            gem update --system
+            gem install bundler
+            bundle install --path vendor/bundle
+      - run:
+          name: Running checks
+          command: bundle exec overcommit --run
```
Comment (bestwebua) [inline .circleci/config.yml]: Try out this trick:
```yml
command: |
    bundle exec overcommit -s
    SKIP=AuthorEmail,AuthorName bundle exec overcommit -r
```

Please add end empty line.

File: .fasterer.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+exclude_paths:
+  - 'vendor/**/*.rb'
+  - 'db/schema.rb'
```
Comment (bestwebua) [inline .fasterer.yml]: Empty line.

File: bin/yarn [hunk 3]
```diff
@@ -1,17 +1,17 @@
 #!/usr/bin/env ruby
 APP_ROOT = File.expand_path('..', __dir__)
 Dir.chdir(APP_ROOT) do
-  yarn = ENV["PATH"].split(File::PATH_SEPARATOR).
-    select { |dir| File.expand_path(dir) != __dir__ }.
-    product(["yarn", "yarn.cmd", "yarn.ps1"]).
-    map { |dir, file| File.expand_path(file, dir) }.
-    find { |file| File.executable?(file) }
+  yarn = ENV['PATH'].split(File::PATH_SEPARATOR)
```
Comment (bestwebua) [inline bin/yarn]: Where is atomic PR for Rails init? There should be no JS in api application.

File: Gemfile [hunk 4]
```diff
@@ -41,5 +41,15 @@ group :test do
   gem 'webdrivers'
```
Comment (bestwebua) [inline Gemfile:25]: For what this dependency?

**Review by bestwebua — CHANGES_REQUESTED** [review]
Better idea is init new rails api project via generator without needless dependencies. For instance:

```bash
rails new your-project-name --api --minimal --database=postgresql -J -T
```

File: Gemfile (hunk 4 above)
Comment (vanyavasylyshyn) [reply Gemfile:25]: This dependency was not removed by cleaning before intitial commit. I'll create another PR with deleting all not needed stuff

**Review by bestwebua — APPROVED** [review]

**Pull request merged**